Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -104,21 +98,105 @@ final class RewardsCollectionViewController: UICollectionViewController {
       .observeForUI()
       .observeValues { [weak self] rewards in
         self?.dataSource.load(rewards: rewards)
+        self?.collectionView.reloadData()
     }
   }
 
+  // MARK: - Private Helpers
+
+  private func configureHiddenScrollView() {
+    _ = self.hiddenPagingScrollView
+      |> \.delegate .~ self
+
+    _ = (self.hiddenPagingScrollView, self.view)
+      |> ksr_addSubviewToParent()
+
+    self.view.sendSubviewToBack(self.hiddenPagingScrollView)
```

Comment:
Would it make sense to create another function helper for inserting subviews? This way we wouldn't need to do 2 operations (add & sending to back)? But can do instead...

```swift
func ksr_insertSubviewToParentAt(index: Int) -> (UIView, UIView) -> (UIView, UIView) {
  return { (subview, parent) in
    parent.insertSubview(subview, at: index)

    return (subview, parent)
  }
}
```

and call it like so

```swift
_ = (self.hiddenPagingScrollView, self.view)
  |> ksr_insertSubviewToParentAt(index: 0)
```